pkg/api/const.go

```diff
@@ -306,6 +306,22 @@ const (
 	DefaultKubernetesCloudProviderRateLimitBucketWrite = DefaultKubernetesCloudProviderRateLimitBucket
 )
 
+// Azure Stack configures all clusters as if they were large clusters.
+const (
+	DefaultAzureStackKubernetesCloudProviderBackoffRetries       = 6
+	DefaultAzureStackKubernetesCloudProviderBackoffJitter        = 1.0
+	DefaultAzureStackKubernetesCloudProviderBackoffDuration      = 5
+	DefaultAzureStackKubernetesCloudProviderBackoffExponent      = 1.5
+	DefaultAzureStackKubernetesCloudProviderRateLimitQPS         = 3.0
+	DefaultAzureStackKubernetesCloudProviderRateLimitQPSWrite    = 3.0
+	DefaultAzureStackKubernetesCloudProviderRateLimitBucket      = 10
+	DefaultAzureStackKubernetesCloudProviderRateLimitBucketWrite = 10
+	DefaultAzureStackKubernetesNodeStatusUpdateFrequency         = "1m"
+	DefaultAzureStackKubernetesCtrlMgrRouteReconciliationPeriod  = "1m"
+	DefaultAzureStackKubernetesCtrlMgrNodeMonitorGracePeriod     = "5m"
+	DefaultAzureStackKubernetesCtrlMgrPodEvictionTimeout         = "1m"
```

Comment:
Today default value is "--pod-eviction-timeout=5m0s". Lets better check why is that value ?.

Reply:
https://github.com/Azure/aks-engine/blob/feecc8b4f6173c68d6ee15792036e5e0faed03c6/pkg/api/const.go#L399

Reply:
FYI, it aligns with the large clusters [guideline](https://github.com/Azure/aks-engine/blob/master/docs/howto/kubernetes-large-clusters.md)

Reply:
The docs is for all K8s version. New behavior is different from 1.14 which is officially supported on Azure Stack Hub.
We should check once with Jack if we need to update the large cluster guideline page. As it might be stale and may need some updates.

Reply:
@jackfrancis is the guideline stale?